int64 reqImage
---
bool result